src/core/storage.tf

```diff
@@ -0,0 +1,40 @@
+resource "azurerm_resource_group" "rg_storage" {
+  name     = format("%s-storage-rg", local.project)
+  location = var.location
+  tags     = var.tags
+}
+
+## Storage account to save blob
+#tfsec:ignore:azure-storage-default-action-deny
+module "selc-blob-storage" {
+  source = "git::https://github.com/pagopa/azurerm.git//storage_account?ref=v1.0.79"
+
+  name                       = replace(format("%s-blobstorage", local.project), "-", "")
```

Comment:
Can we rename this storage? I think that this is a special storage so we can rename for example as `contracts`